legion/legion/serving/pyserve.py

```diff
@@ -86,6 +97,26 @@ def model_invoke(model_id):
     return legion.http.prepare_response(output)
 
 
+@blueprint.route(SERVE_BATCH.format(model_id='<model_id>'), methods=['POST'])
+def model_batch(model_id):
+    """
+    Call model for calculation in batch model
+
+    :param model_id: model name
```

Comment:
You pass a Model Name to validate_model_id function. It confuses.

Reply:
why?